cloudify_marketplace/resources/cloudify_security_reconfiguration

```diff
@@ -0,0 +1,267 @@
+#! /usr/bin/env python
+import json
+import logging
+import logging.handlers
+import os
+import re
+import shutil
+import subprocess
+import time
+import yaml
```

Comment:
No, I think we're trying to follow PEP8's (?) import guide-
stdlib
stuff from this module/package/whatever
third party (which cloudify technically is as far as this is concerned)

I did some research last year, but I don't think there are any completely reliable checkers for it.